src/osm_configurator/model/project/configuration/calculation_method_of_area_enum.py

```diff
@@ -20,5 +20,11 @@ def get_calculation_method(self):
         Returns:
             str: The name of the Calculation Method.
         """
-        pass
+        return self.value
+
+    def equals(name: str) -> CalculationMethodOfArea | None:
```

Comment:
better name or descirbe in pydoc what this method does.
A "normal" equals method would return a boolean.